### PR TITLE
Add batch getRecordByIds call to request.js

### LINKED ISSUE #20: Add a batch `findRecordsByIds` function
Currently if you have an array of record IDs, you have to make a unique `findRecordById` call for each one. Can we make a function that takes in an array of IDs and only makes one fcall?

### PR DESCRIPTION
**Pull request by aivantg**
## What's in this PR

- Added a function to `requests.js` that gets all records with ID in a given array of ID. 

Creates a function like this: 
```
export const getOwnerByIds = async ids => {
  const formula = `OR(${ids.reduce((f, id) => `${f}, {ID}='${id}'`)})`;
  return getAllRecords(Tables.Owner, formula);
};
```

## Related Issues

closes #20

### CODE REVIEW

File: lib/snippets.js [hunk 1]
```diff
@@ -88,6 +88,11 @@ export const get${tableName}ById = async id => {
   return getRecordById(Tables.${tableName}, id);
 };
 
+export const get${tableName}ByIds = async ids => {
```
Comment (annieyro) [inline lib/snippets.js]: Real nitpick: call `pluralize` here - (you're getting many owners by their IDs, right?)

Comment (aivantg) [reply lib/snippets.js]: great point

**Review by annieyro — APPROVED** [review]
Pluralize the function name! Also, you might want to add a comment somewhere (README? inline?) about the specific use case (as in batch record ID? bc I don't think it's super intuitive by the name of the function)

File: .env.example [hunk 2]
```diff
@@ -1,3 +1,4 @@
 AIRTABLE_BASE_ID=
 AIRTABLE_EMAIL=
 AIRTABLE_PASSWORD=
+REACT_APP_AIRTABLE_API_KEY=
```
Comment (annieyro) [inline .env.example]: Noo why is this back

**Pull request merged**